aks-node-controller/parser/helper.go

```diff
@@ -613,6 +609,68 @@ func getServicePrincipalFileContent(authConfig *aksnodeconfigv1.AuthConfig) stri
 	return base64.StdEncoding.EncodeToString([]byte(authConfig.GetServicePrincipalSecret()))
 }
 
+func getKubeletFlags(kubeletConfig *aksnodeconfigv1.KubeletConfig) string {
+	return createSortedKeyValuePairs(kubeletConfig.GetKubeletFlags(), " ")
+}
+
+// pass the type of v as an input arguem
```

Comment:
typo: argument

Reply:
I also think we should disable godot linter...

Reply:
updated!